src/AutoRest.CSharp.V3/ClientModels/FrameworkTypeReferenceFormat.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace AutoRest.CSharp.V3.ClientModels
+{
+    internal enum FrameworkTypeReferenceFormat
```

Comment:
Can we share the SerializationFormat enum?

Reply:
I had mentioned this before... when it was just a header format. I'll reuse it. I wrote these changes before that change.

Reply:
Fixed.